utils/data_channel_rpc.go

```diff
@@ -64,9 +64,9 @@ func (e *DataChannelRpcError) PsrpcError() psrpc.Error {
 	case DataChannelRpcApplicationError:
 		return psrpc.NewErrorf(psrpc.Internal, e.Message, "data", e.Data)
 	case DataChannelRpcConnectionTimeout:
-		return psrpc.NewErrorf(psrpc.DeadlineExceeded, e.Message, "data", e.Data)
+		return psrpc.NewErrorf(psrpc.Canceled, e.Message, "data", e.Data)
 	case DataChannelRpcResponseTimeout:
-		return psrpc.NewErrorf(psrpc.DeadlineExceeded, e.Message, "data", e.Data)
```

Comment:
how does that translate to API error?  Feels like `DeadlineExceeded` is the right one and that will translate to `Timeout`